Source/WebKit/Platform/spi/Cocoa/BlockSPI.h

```diff
@@ -23,7 +23,8 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-#if USE(APPLE_INTERNAL_SDK)
+// FIXME: COMPILER(APPLE_CLANG) is needed until <rdar://120571425> is fixed.
+#if USE(APPLE_INTERNAL_SDK) && COMPILER(APPLE_CLANG)
 
 #import <Block_private.h>
```

Comment:
If we can compile properly without this include, perhaps it can be just removed?

Reply:
> If we can compile properly without this include, perhaps it can be just removed?

That's basically true for any internal/private header.  :)

We've always preferred the private header over redeclaring a function when compiling with the internal SDK.

The `<Block_private.h>` header is declaring this function (which is re-declared for the public SDK):

```
BLOCK_EXPORT const char* _Block_signature(void*);
```

(Not sure why GitHub won't let you expand the rest of the file, though.)

Reply:
Makes sense, I didn't see that it was redeclared below.